lib/WTSI/NPG/HTS/PacBio/Sequel/AnalysisPublisher.pm

```diff
@@ -125,12 +137,12 @@ sub publish_files {
 =cut
 
 sub publish_sequence_files {
-  my ($self,$format) = @_;
+  my ($self, $format, $subdironly) = @_;
```

Comment:
As this is available as an attribute (`$self->is_oninstrument`) I don't think it needs to be a new param. Why not test the attribute directly, within the method? Having both options opens up the possibility of bugs where the attribute is set, but sometimes it's accidentally not passed as a param. Relying on the attribute means that all methods will behave in the same manner, it it's set.

if it really needs to be a param, the POD needs updating because it currently doesn't mention it.